web/src/pages/profile.tsx

```diff
@@ -487,6 +489,7 @@ const ProfilePage = (props: Props): ReactElement => {
         </ProfileField>
         <ProfileField fieldName="industryId">
```

Comment:
Shouldn't we have 3 additions to this file for Poppy, Oscar and Dakota Nexus? Where is Oscar? Am I blind?

Reply:
The ticket is only for Poppy and Dakota Nexus because those are the only ones with industries, Oscar has sectors so this isn't applicable for it.

Reply:
Gotcha, weird that the ticket calls out Oscar then